full_name.rb

```diff
@@ -0,0 +1,25 @@
+
+#Write a program which asks for a person's first name, then middle, then last.  
+#It should store each of these parts in an array. 
+#Finally, it should greet the person using their full name.  
+#Call the program full_name.rb.
+
+greet_person = []
+def ask(question)
+    print question + ""
+end
+    f_name = ask("Enter your first name ")
+    f_name = gets.chomp
+    greet_person.push(f_name)
+
+    m_name = ask("Enter your middle name ")
+    m_name = gets.chomp
+    greet_person.push(m_name)
+
+    l_name = ask("Enter your last name ")
+    l_name = gets.chomp
+    greet_person.push(l_name)
+
+    full_name = greet_person.join(' ')
+    print "Greetings #{full_name}!"
```

Comment:
Confusing indentation. I would look better this way
![Screenshot from 2020-08-28 11-48-02](https://user-images.githubusercontent.com/47392623/91604919-69b84580-e924-11ea-8369-546364aa73de.png)



This code is good. It is important to have some sort of validation with the inputs (I would say mainly in first name and last name) because, if we leave them blank, this is the final message:
`Greetings   !`

We can add some sort of conditional. If the user doesn't enter anything either first or last name, the program returns a message saying something like "This field is required". This is not mandatory to do, however, it is a good practice always making "full proof" programs.